### ISSUE TITLE
completely straight track invisible

### ISSUE THREAD

**Issue by fabi321** [Maproom/qmapshack#580]
### Describe the bug
A track that only is a perfectly straight line, is invisible in the map view of QMapShack

### What have you done to circle down the problem?
All other tracks of the same gpx file are perfectly visible, and the line appears in the track overview

### To Reproduce
Load a gpx track containing
```
  <trk>
    <trkseg>
      <trkpt lat="-0.35099682625000006" lon="-0.05026035450000001">
        <ele>2.57</ele>
      </trkpt>
      <trkpt lat="-0.35099682625000006" lon="-0.049964866850000005">
        <ele>2.57</ele>
      </trkpt>
      <trkpt lat="-0.35099682625000006" lon="-0.04989109447">
        <ele>2.57</ele>
      </trkpt>
      <trkpt lat="-0.35099682625000006" lon="-0.04880602623">
        <ele>2.57</ele>
      </trkpt>
    </trkseg>
  </trk>
```

### Expected behavior
It should be visible as a straight line

### Screenshots
![image](https://user-images.githubusercontent.com/42294590/213788877-48e0fa60-aad6-427f-80ed-22de14fc60bd.png)
The line is supposed to be visible above the line that is curling down, starting at the (kinda) visible joint to the left.

### Attachments
[full_gpx_file.zip](https://github.com/Maproom/qmapshack/files/10469645/ade.zip)

### Desktop

 - OS: Ubuntu
 - QMapShack Version: 1.16.1
 - Window Manager: KDE

**Comment by fabi321** [comment]
This bug only seems to affect horizontal lines, vertical lines are working as expected

**Comment by wthaem** [comment]
Snapshot on Windows: all 3 tracks visible.

![TRK_Visible](https://user-images.githubusercontent.com/55309218/213859472-c3387379-88a1-4c17-9097-1f472cc49934.jpg)

**Comment by frankystone** [comment]
Can't also reproduce. But i am using current dev-version. Each track with different colors:

![tracks](https://user-images.githubusercontent.com/6730556/213859916-9a36b5ab-56a7-4f93-abbc-b503bfe6183b.png)

**Comment by kaskasapakte** [comment]
Should be fixed in the nest release, see [#517](https://github.com/Maproom/qmapshack/issues/517).

**Comment by fabi321** [comment]
That issue slipped through my search, thx!
